lib/charms/postgresql_k8s/v1/postgresql.py

```diff
@@ -311,6 +312,7 @@ def create_user(
                     user_definition += " CREATEDB"
                 if privileges:
                     user_definition += f" {' '.join(privileges)}"
+                cursor.execute(SQL("RESET ROLE;"))
```

Comment:
PgBouncer needs to reset its user from `pgbouncer_owner` to `relation-X` (which is a superuser) to call `SET LOCAL log_statement = 'none';`.